bids_prov/spm_load_config.py

```diff
@@ -38,12 +38,15 @@ def get_empty_graph(context_url=CONTEXT_URL):
         "records": {
             "prov:Agent": [
                 {
-                    "@id": "RRID:SCR_007037",  # TODO query for version
+                    "@id": "exampleAgentID",
+                    "RRID": "RRID:SCR_007037",
                     "@type": "prov:SoftwareAgent",
                     "label": "SPM",
+                    "version": "dev"
```

Comment:
The version of the software should be passed as an argument, I would recommend using "SPM12 r7224" for our tests